pages/en/lb4/Download-examples.md

```diff
@@ -0,0 +1,43 @@
+---
+lang: en
+title: 'Download examples'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Download-examples.html
+summary:
+---
+
+### Synopsis
+
+Downloads a LoopBack example project from our GitHub monorepo.
```

Comment:
Would it be good to include the loopback-next repo link to `our GitHub monorepo`?